core/modules/loader/CentralWorker.h

```diff
@@ -253,14 +254,14 @@ class CentralWorker : public Central {
 
     std::atomic<bool> _shiftAsClientInProgress{false}; ///< True when shifting to or from right neighbor.
 
-    /// Shift if a node has 10% more than it's neighbor TODO config file DM-16652
-    double _thresholdNeighborShift{1.10};
+    /// Shift if a node has % more than it's neighbor (1.1 would be 10% more than neighbor)
+    double _thresholdNeighborShift;
```

Comment:
it appears the 1.1 this refers to has been removed. Maybe you still want to say something like "the percentage threshold is expressed as a decimal, so e.g. 110% of the neighbor would be 1.10"